### DISCUSSION TITLE
32 bits support

### DISCUSSION THREAD

**Discussion by j-jorge** [axmolengine/axmol#2283] [General]
Hello, during the configuration step of my Android project a message printed via CMake indicates that 32 bits (armeabi-v7a) is going to be dropped soon.

I have a Nexus 4 on which I install my game. If 32 bits support is dropped I won't be able to use my app on this device. Moreover I usually communicate in a community where people appreciate software that runs on older devices, for ecological, economic, and ethical reasons.

Is there a necessity to drop this architecture?

**Comment by mehdisalescale** [comment]


**Reply by danialias** [reply]
Is this some kind of spam?

**Reply by rh101** [reply]
Reported and deleted the post.
